plugins/lsp/language-servers/_mk-lsp.nix

```
@@ -55,6 +55,15 @@ in
         lib.nixvim.mkMaybeUnpackagedOption "plugins.lsp.servers.${name}.package" pkgs name
           package;
 
+      packageFallback = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          When enabled, the language server package will be added to the end of `PATH` (--suffix) instead of the beginning (--prefix).
+          This can be useful if you want local versions of the language server (e.g. from a devshell) to override the nixvim version.
```

```suggestion

          This can be useful if you want local versions of the language server (e.g. from a devshell) to override the nixvim version.
```

Need a fully blank line (`\n\n`) to separate markdown paragraphs, otherwise markdown will join the lines into a single line.